lib/Ast.ml

```diff
@@ -2268,6 +2268,13 @@ end = struct
       , {pexp_desc= Pexp_construct _ | Pexp_variant _; _} )
       when e == exp ->
         true
+    | ( Exp {pexp_desc= Pexp_apply (e, _ :: _); _}
+      , {pexp_desc= Pexp_apply (op1, args); pexp_attributes; _} )
+      when e == exp && Exp.is_prefix op1 -> (
+      match (args, pexp_attributes) with
+      | _, _ :: _ -> true
+      | [(Nolabel, _)], _ -> false
```

Comment:
Am I right that this case is not explicitly covered by the new test cases?

Reply:
Not in the new tests but it's covered by apply.ml, source.ml and js_source.ml